float32[] goal
uint8 type
float32 speed
float32 acceleration
float32 tolerance
float32 delay
---
bool response